packages/nodes-base/nodes/WhatsApp/WhatsApp.node.ts

```diff
@@ -50,6 +61,53 @@ export class WhatsApp implements INodeType {
 			...mediaFields,
 			...messageTypeFields,
 			...mediaTypeFields,
+			...getSendAndWaitProperties([], 'message', undefined, {
+				noButtonStyle: true,
+				defaultApproveLabel: '✓ Approve',
+				defaultDisapproveLabel: '✗ Decline',
+			}).filter((p) => p.name !== 'subject'),
 		],
 	};
+
+	webhook = sendAndWaitWebhook;
+
+	customOperations = {
+		message: {
+			async [SEND_AND_WAIT_OPERATION](this: IExecuteFunctions) {
+				try {
+					const phoneNumberId = this.getNodeParameter('phoneNumberId', 0) as string;
+					const recipientPhoneNumber = (
+						this.getNodeParameter('recipientPhoneNumber', 0) as string
+					).replace(/[\-\(\)\+]/g, '');
```

Comment:
What does this regex do? Can we extract it into it's own function with proper name and tests

Reply:
I don't have much context about this, but this is used to cleanup `recipientPhoneNumber` parameter in `send` operation of that node, so make sense to do this here as well

Reply:
Let's extract it into a separate method called `cleanupPhoneNumber` so it communicates what it does without someone reading this code having to guess